src/views/blocks/tickets.php

```diff
@@ -73,56 +75,81 @@
+			'content'         => __( 'The numbers below include tickets for this event already in your cart. Clicking "Get Tickets" will allow you to edit any existing attendee information as well as change ticket quantities.', 'event-tickets' ),
+		]
+	);
+
+	$this->template(
+		'blocks/tickets/commerce/fields',
+		[
+			'provider'    => $provider,
+			'provider_id' => $provider_id,
 		]
-	); ?>
+	);
+
+	if ( $has_tickets_on_sale ) :
+		foreach ( $tickets_on_sale as $key => $ticket ) :
+			$ticket_symbol = $currency->get_currency_symbol( $ticket->ID, true );
+
+			$this->template(
+				'blocks/tickets/item',
+				[
+					'ticket'          => $ticket,
+					'key'             => $key,
+					'currency_symbol' => $ticket_symbol,
+				]
+			);
+		endforeach;
 
-	<?php $this->template( 'blocks/tickets/commerce/fields', [ 'provider' => $provider, 'provider_id' => $provider_id ] ); ?>
-	<?php if ( $has_tickets_on_sale ) : ?>
-		<?php foreach ( $tickets_on_sale as $key => $ticket ) : ?>
-			<?php $ticket_symbol = $currency->get_currency_symbol( $ticket->ID, true ); ?>
-			<?php $this->template( 'blocks/tickets/item', [ 'ticket' => $ticket, 'key' => $key, 'currency_symbol' => $ticket_symbol ] ); ?>
-		<?php endforeach; ?>
-		<?php
 		// We're assuming that all the currency is the same here.
-		$currency_symbol     = $currency->get_currency_symbol( $ticket->ID, true );
-		$this->template( 'blocks/tickets/footer', [ 'tickets' => $tickets, 'currency_symbol' => $currency_symbol ] );
-		?>
-	<?php else : ?>
-		<?php $this->template( 'blocks/tickets/item-inactive', [ 'is_sale_past' => $is_sale_past ] ); ?>
-	<?php endif; ?>
-	<?php
-		ob_start();
-		/**
-		 * Allows filtering of extra classes used on the tickets-block loader
-		 *
-		 * @since  4.11.0
-		 *
-		 * @param  array $classes The array of classes that will be filtered.
-		 */
-		$loader_classes = apply_filters( 'tribe_tickets_block_loader_classes', [ 'tribe-tickets-loader__tickets-block' ] );
-		include Tribe__Tickets__Templates::get_template_hierarchy( 'components/loader.php' );
-		$html = ob_get_contents();
-		ob_end_clean();
-		echo $html;
+		$currency_symbol = $currency->get_currency_symbol( $ticket->ID, true );
+		$this->template(
+			'blocks/tickets/footer',
+			[
+				'tickets'         => $tickets,
+				'currency_symbol' => $currency_symbol,
+			]
+		);
+	else :
+		$this->template( 'blocks/tickets/item-inactive', [ 'is_sale_past' => $is_sale_past ] );
+	endif;
+
+	ob_start();
+	/**
+	 * Allows filtering of extra classes used on the tickets-block loader
+	 *
+	 * @since  4.11.0
+	 *
+	 * @param  array $classes The array of classes that will be filtered.
+	 */
+	$loader_classes = apply_filters( 'tribe_tickets_block_loader_classes', [ 'tribe-tickets-loader__tickets-block' ] );
+	include Tribe__Tickets__Templates::get_template_hierarchy( 'components/loader.php' );
```

Comment:
:warning: File is being unconditionally included; use "require" instead

<sub>hash: 21becf9662052f8ecc526da4041d31b0</sub>